drivers/misc/misc_ivshmem_shell.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <shell/shell.h>
+#include <stdlib.h>
+#include <drivers/misc/ivshmem.h>
+static const struct device *ivshmem;
+
+static bool get_ivshmem(const struct shell *shell)
+{
+	if (ivshmem == NULL) {
+		ivshmem = device_get_binding(CONFIG_IVSHMEM_DEV_NAME);
+		if (!ivshmem) {
+			shell_error(shell, "IVshmem device cannot be found\n");
```

Comment:
The macro `shell_error` already adds a new line. If you want to have 2 lines break it is fine.

Reply:
no indeed, it's an error. probably got mixed up with shell_fprintf